tensornetwork/visualization/graphviz.py

```diff
@@ -40,36 +44,47 @@ def to_graphviz(net: network.TensorNetwork,
   if graph is None:
     #pylint: disable=no-member
     graph = graphviz.Graph('G', engine=engine)
-  for node in net.nodes_set:
-    if not node.name.startswith("__") or include_all_names:
-      label = node.name
-    else:
-      label = ""
-    graph.node(str(node.signature), label=label)
-  seen_edges = set()
-  for node in net.nodes_set:
-    for i, edge in enumerate(node.edges):
-      if edge in seen_edges:
-        continue
-      seen_edges.add(edge)
-      if not edge.name.startswith("__") or include_all_names:
-        edge_label = edge.name
-      else:
-        edge_label = ""
-      if edge.is_dangling():
-        # We need to create an invisible node for the dangling edge
-        # to connect to.
-        graph.node(
-            "{}_{}".format(node.signature, i),
-            label="",
-            _attributes={"style": "invis"})
-        graph.edge(
-            "{}_{}".format(node.signature, i),
-            str(node.signature),
-            label=edge_label)
+  if not net:
+    for node in net.nodes_set:
+      if not node.name.startswith("__") or include_all_names:
+        label = node.name
```

Comment:
I don't understand this part.

If the network is `None`, why would you iterate over `net.nodes_set`?